services/storage/src/simcore_service_storage/simcore_s3_dsm.py

```diff
@@ -240,21 +238,21 @@ async def list_files(  # noqa C901
         return data
 
     async def get_file(self, user_id: UserID, file_id: StorageFileID) -> FileMetaData:
-        async with self.engine.acquire() as conn, conn.begin():
+        async with self.engine.acquire() as conn:
             can: AccessRights = await get_file_access_rights(
                 conn, int(user_id), file_id
             )
-            if can.read:
-                fmd: FileMetaDataAtDB = await db_file_meta_data.get(
-                    conn, parse_obj_as(SimcoreS3FileID, file_id)
-                )
-                if is_file_entry_valid(fmd):
-                    return convert_db_to_model(fmd)
-                fmd = await self._update_database_from_storage(conn, fmd)
-                return convert_db_to_model(fmd)
+            if not can.read:
+                raise FileAccessRightError(access_right="read", file_id=file_id)
 
-            _logger.debug("User %s cannot read file %s", user_id, file_id)
-            raise FileAccessRightError(access_right="read", file_id=file_id)
+            fmd = await db_file_meta_data.get(
+                conn, parse_obj_as(SimcoreS3FileID, file_id)
+            )
+        if is_file_entry_valid(fmd):
+            return convert_db_to_model(fmd)
```

Comment:
every time i see these `convert_db_to_model` I get a bit down :-)

Reply:
well that was pre-pydantic. but this is outside this PR focus